frontend/src/components/Item/ItemMenu.vue

```diff
@@ -284,6 +303,28 @@ function onActivatorClick(): void {
   positionY.value = undefined;
 }
 
+/**
+ * Handle deletion of the item
+ */
+deleteDialog.onConfirm(async () => {
+  if (!menuProps.item.Id) {
+    useSnackbar(t('errors.anErrorHappened'), 'error');
+
+    return;
+  }
+
+  try {
+    await remote.sdk.newUserApi(getLibraryApi).deleteItem({
+      itemId: menuProps.item.Id
+    });
+    router.replace('/');
```

Comment:
I guess this is only needed if we're in the page of an item? We can check if the route contains the item id before doing this I guess?